qiita_pet/templates/study_ajax/prep_summary.html

```diff
@@ -353,7 +353,10 @@
 <!-- #samples and #columns -->
 <div class="row">
   <div class="col-md-12">
-    There are <b>{{num_samples}}</b> samples and <b>{{num_columns}}</b> columns in this preparation.
+    There are <b>{{num_samples}}</b> samples and <b>{{num_columns}}</b> columns in this preparation.</br>
+    {% if ebi_experiment_accessions %}
+      <b>EBI status: </b> Submitted
```

Comment:
Got it.

On (Apr-20-16| 9:36), Antonio Gonzalez wrote:
> > @@ -353,7 +353,10 @@
> >  <!-- #samples and #columns -->
> >  <div class="row">
> >    <div class="col-md-12">
> > -    There are <b>{{num_samples}}</b> samples and <b>{{num_columns}}</b> columns in this preparation.
> > +    There are <b>{{num_samples}}</b> samples and <b>{{num_columns}}</b> columns in this preparation.</br>
> > +    {% if ebi_experiment_accessions %}
> > +      <b>EBI status: </b> Submitted
> 
> Not really, this is for preps. This is, the study is what's being submitted/updated and the prep is only submitted/not-submitted and we know is submitted if the samples have ebi_experiment_accessions.
> 
> ---
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/1804/files/854af66518166469335927275d9319f9a7ef5a1e#r60443512